code/modules/antagonists/heretic/knowledge/cosmic_lore.dm

```diff
@@ -68,8 +67,7 @@
 	desc = "Grants you Cosmic Runes, a spell that creates two runes linked with eachother for easy teleportation. \
 		Only the entity activating the rune will get transported, and it can be used by anyone without a star mark. \
 		However, people with a star mark will get transported along with another person using the rune."
-	gain_text = "When day came, the Sleeper got lost. \
-		The sun outshone the stars, so he lost his guide."
+	gain_text = "A sigil appeared below me, I lowered my face to it and heard distant roars, I flung myself in and saw a beast behind dense clouds of gas."
```

Comment:
```suggestion
	gain_text = "A sigil appeared below me, I lowered my face to it and heard distant roars. I flung myself in and saw a beast behind dense clouds of gas."
```
I'll be honest I also think the repeated use of the word "beast" here is kind of uninspired and not very mystical. Can we call it something else? 
Even just a "figure" is more interesting to me than "beast".